packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -1,5 +1,5 @@
 import { BufferComposer, ComposableBuffer } from '@defichain/jellyfish-buffer'
-import { CTokenBalance, CTokenBalanceVarInt, TokenBalance, TokenBalanceVarInt } from './dftx_balance'
+import { CTokenBalance, TokenBalance } from './dftx_balance'
```

Comment:
here i think `dftx_loans` need support the removed vault function as well
try export import here will do

Reply:
I already exported vaults functions via https://github.com/DeFiCh/jellyfish/pull/1252/files#diff-2f82061c33af424e1be7b95e01271a0a9c3351225da34587cb8db8ca3764c604R271
 Its a `*` export so should be exporting everything from vault as well.